python/astro_metadata_translator/translator.py

```diff
@@ -1113,6 +1113,22 @@ def to_has_simulated_content(self) -> bool:
         """
         return False
 
+    @cache_translation
+    def to_focus_z(self) -> u.Quantity:
+        """Return a default defocal distance of 0.0 mm if there is no
+        keyword for defocal distance in the header. The default
+        keyword for defocal distance is ``FOCUSZ``.
+
+        Returns
+        -------
+        focus_z: `astropy.units.Quantity`
+            The defocal distance from header or the 0.0mm default
+        """
+        if self.is_key_ok("FOCUSZ"):
```

Comment:
I think I'm still inclined to drop this check because `FOCUSZ` is not part of any standard.  The base implementation should return 0.0mm and not look at any headers.